PRIVACY.md

```diff
@@ -268,6 +270,7 @@ This is a sample blob of the aggregate metrics described above:
         "numFindings": 7,
         "numIgnored": 3,
         "features": ["language/python", "option/deep", "option/no-git-ignore", "key/metavariable-comparison"]
+        "unsupportedExts": {".txt": 3, ".erb": 2, ".java": 5}
```

Comment:
This is a nit but we do support `.java`. Even though this is just an example, I could see someone reading this and being confused. Shouldn't be too hard to dig up a language we don't support

Reply:
One thing with this feature is that even if we do support a language, if the user doesn't have any rules selected that scan it, it will show up here---so this could be a real output. But I am happy to change it to be a different language. However, I realized that I misunderstood part of the requirements so am transitioning this to be in the scan metadata---I think that makes this section irrelevant. Sorry to waste your reviewing time